crates/telio-model/src/api_config.rs

```diff
@@ -219,12 +219,16 @@ pub struct Features {
     pub direct: Option<FeatureDirect>,
     /// Test environment (natlab) requires binding feature disabled
     /// TODO: Remove it once mac integration tests support the binding mechanism
-    #[cfg(any(target_os = "macos", feature = "pretend_to_be_macos"))]
+    #[serde(default = "default_is_test_env")]
     pub is_test_env: Option<bool>,
     /// Derp server specific configuration
     pub derp: Option<FeatureDerp>,
 }
 
+fn default_is_test_env() -> Option<bool> {
+    Some(true)
+}
```

Comment:
I don't fully remember why this defaults to true, but I remember trying to just do the type default (back then false, because it wasn't yet an option) and that broke stuff in tests

Reply:
I can do that, though that `cfg` attribute doesn't persist across crate boundaries so other crates who depend on `telio-model` would get the `not(test)` branch all the time

Reply:
IMO it should default to false. In current form it will be defaulted to true also in production which is incorrect. It should default to false and be changed to true only via config during natlab tests

Reply:
I don't know why it became an optional in the first place, but opinion of defaulting to `None`?

Reply:
It should also be fine, and maybe even the cleanest option